tests/phpunit/CRM/Utils/MoneyTest.php

```diff
@@ -72,13 +72,6 @@ public function testFormatLocaleNumericRoundedByCurrencyEuroThousand() {
     $this->setCurrencySeparators(',');
   }
 
-  /**
-   * Test that using the space character as a currency works
-   */
-  public function testSpaceCurrency() {
-    $this->assertEquals('  8,950.37', CRM_Utils_Money::format(8950.37, ' '));
-  }
```

Comment:
Wow, using a space as currency is [bonkers](https://en.wikipedia.org/wiki/Whitespace_(programming_language)). Big 🎉 on a clearer contract.

From a skim, it looks like `CRM_Utils_Money::format()` now emits a warning but doesn't change behavior? Considering that `CRM_Utils_Money::format` shows up a lot in `universe`, that seems like good policy. :+1:

Just thinking a bit about the next step in the phase-out process -- eventually, the behavior goes from "deprecated" to "broken/unsupported". Since the test has been written, maybe keeping it would ensure we get a signal when it becomes broken/unsupported... so then we can update the error-message/release-notes accordingly. Maybe prevent confusion by renaming the test (`testSpaceCurrency_Deprecated()`?) or adding a comment about the when/why of deprecation.

If the deprecation notice is mucking with the test, then maybe [one of these](https://stackoverflow.com/questions/8412746/ignoring-the-php-warnings-in-phpunit) would help?